app/src/main/java/com/example/loic/comics_app_android/presentation/ui/activity/ComicListActivity.java

```diff
@@ -0,0 +1,85 @@
+package com.example.loic.comics_app_android.presentation.ui.activity;
+
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.RecyclerView;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+
+import com.example.loic.comics_app_android.R;
+import com.example.loic.comics_app_android.data.model.ResultsItem;
+import com.example.loic.comics_app_android.presentation.ui.adapter.RecyclerComicsListAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ComicListActivity extends AppCompatActivity {
+
+    private RecyclerView mRecyclerView;
+    private RecyclerView.Adapter mAdapter;
+    private RecyclerView.LayoutManager mLayoutManager;
+
+    //DEBUG
+    private List<ResultsItem> comicList = new ArrayList<>();
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_comics_list);
+
+        this.setTitle("Liste de comics");
+
+        mRecyclerView = (RecyclerView) findViewById(R.id.comics_list_rv);
+        mRecyclerView.setHasFixedSize(true); //Pas de changement de size du layout -> fixe pour les perfs
+
+        mLayoutManager = new LinearLayoutManager(this);
+        mRecyclerView.setLayoutManager(mLayoutManager);
```

Comment:
Je pense plutôt qu'on devrais set le LayoutManager comme ceci :  `mRecyclerView.setLayoutManager(new LinearLayoutManager(this))`

Reply:
✔️